build/runbenchmarks.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$3" = "" ]]
+then
+  echo "Usage: runbenchmarks.sh <repo-directory> <results-directory> <target-framework> [target-framework...]"
+  echo "e.g. runbenchmarks.sj benchmark-tmp benchmark-results netcoreapp1.1 net45"
+  exit 1
+fi
+
+declare -r REPO=$1
+declare -r RESULTS=$(realpath $2)
+
+shift 2
+
+rm -rf $REPO
+git clone https://github.com/nodatime/nodatime.git $REPO --depth=1
+
+declare -r COMMIT=$(git -C $REPO rev-parse HEAD)
+
+if [[ -d $RESULTS/$COMMIT ]]
```

Comment:
I assume RESULTS can't contain spaces, otherwise we're going to have a bad time :)